src/java/org/apache/cassandra/config/EncryptionOptions.java

```diff
@@ -305,7 +306,7 @@ private void ensureConfigNotApplied()
      *
      * @return if the channel should be encrypted
      */
-    public Boolean isEnabled() {
+    public Boolean getEnabled() {
```

Comment:
in java's POJO logic `is` is for `boolean` only, so `Boolean` doesn't get picked up and expects `get` prefix

Reply:
nit: Might as well bump the brace to the next line while we're here?

Reply:
done